models/er.py

```diff
@@ -43,7 +47,12 @@ def observe(self, inputs, labels, not_aug_inputs):
         loss.backward()
         self.opt.step()
 
-        self.buffer.add_data(examples=not_aug_inputs,
+        if original_targets is not None:
+            self.buffer.add_data(examples=not_aug_inputs,
+                             labels=labels[:real_batch_size],
+                             original_labels=original_targets[:real_batch_size])
+        else: 
+            self.buffer.add_data(examples=not_aug_inputs,
                              labels=labels[:real_batch_size])
```

Comment:
instead putting of whole code inside if you can do:
original_targets = original_targets[:real_batch_size] if original_targets is not None
self.buffer.add_data(examples=not_aug_inputs,
                             labels=labels[:real_batch_size],
                             original_labels=original_targets)

original_labels in add_data is by default None, so you will be fine with it